### DISCUSSION TITLE
vcpkg command line allow space around feature names

### DISCUSSION THREAD

**Discussion by linquize** [microsoft/vcpkg#17822] [Ideas]
Currently vcpkg command line does not allow space around feature names.

Valid: `vcpkg install package1[feature1,featue2,feature3]`
Invalid: `vcpkg install package1[feature1, featue2, feature3]`

I suggest to normalize command line argument  `[feature1, featue2, feature3]` to `[feature1,featue2,feature3]`

**Comment by strega-nil** [comment]
I don't think anyone on the team right now has time for this feature; do you want to try adding it? We should be able to give you pointers and help 😄
